### DISCUSSION TITLE
vban applications?

### DISCUSSION THREAD

**Discussion by a-singer** [onyx-and-iris/nvda-voicemeeter#5] [Q&A] [answered]
First and foremost, let me thank you for this, I have wanted to use VB for years and try every little while with no success. I really appreciate your work. I notice on the releases page that there is basic, banana, and potato. The VBAN apps don't seem to be listed. Is it possible to use the project with any of these programs such, for example, as VBANSpot? Thanks again.

**Comment by onyx-and-iris** [comment, marked as answer]
Hello, thanks for checking out the app!

Unfortunately this app only supports the main Voicemeeter GUI. This application is using Voicemeter's Remote API which doesn't seem to have any functions related to VBAN Spot.

I think for an accessible version of VBAN Spot a separate application would be required. Probably some GUI interface implementing the VBAN protocol. I see on Github a few people have made applications for sending/receiving VBAN streams but I can't comment on their accessibility.

That's the best I can tell you, I hope it's helpful nonetheless.
